tests/snappy_pipeline/workflows/test_workflows_abstract.py

```diff
@@ -180,6 +180,7 @@ def default_config_yaml(cls):
                 r"""
                 step_config:
                   dummy:
+                    path_link_in: ""
```

Comment:
Should this be here? As I understand the dummy step it should be agnostic in regards to linkin requirements.

Reply:
Also, here you are testing `dummy_generic_step`, I'd include a similar test for `dummy_generic_step_from_trimmed` (or something that effect).